zuul-core/dependencies.lock

```diff
@@ -238,37 +238,37 @@
             "project": true
         },
         "io.netty:netty-bom": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-buffer": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-codec-haproxy": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-codec-http": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-codec-http2": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-common": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-handler": {
-            "locked": "4.1.69.Final"
+            "locked": "4.1.71.Final"
         },
         "io.netty:netty-tcnative-boringssl-static": {
```

Comment:
I'm trying to check why we're not pulling in the latest tcnative version here. I'd expect `2.0.46`